RandLAPACK/comps/rl_util.hh

```diff
@@ -355,14 +368,46 @@ void gen_exp_mat(
     }
 }
 
+/// Find the condition number of a given matrix A.
+template <typename T>
+T cond_num_check(
+    int64_t m,
+    int64_t n,
+    const std::vector<T>& A,
+    std::vector<T>& A_cpy,
+    std::vector<T>& s,
+    bool verbosity
```

Comment:
replace ``verbosity'' with ``verbose``. The former word describes how verbose something is (which is not a binary) while the latter says whether or not something is verbose.

Reply:
Fixed.